Foundation/NSCFBoolean.swift

```diff
@@ -78,7 +78,10 @@ internal class __NSCFBoolean : NSNumber {
     override var objCType: UnsafePointer<Int8> {
         // This must never be fixed to be "B", although that would
         // cause correct old-style archiving when this is unarchived.
-        return UnsafePointer<Int8>(bitPattern: UInt(_NSSimpleObjCType.Char.rawValue.value))!
+        func _objCType(_ staticString: StaticString) -> UnsafePointer<Int8> {
```

Comment:
As I tested, `utf8Start` points directly cstring in `__TEXT,__cstring,cstring_literals` whether `StaticString` is instantiated as static variable or function parameter.